test/benchmarks/material/button-harness/constants.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export const USE_BENCHPRESS = false;
```

Comment:
Can you add a comment on when this should/can be adjusted? Why does it even exist? what's the use-case for not using benchpress in protractor? 

Also, should the non-benchpress tests in both protractor and testbed also perform multiple samples? We do this in framework and that would help with stabilizing the measured times?

Reply:
For testbed, I have added multiple samples. For protractor, there's no way to average using `console.time`

Reply:
You might be able to use `process.hrtime` in NodeJS. That might be perfect for our needs here.

Reply:
That works